src/app.ts

```diff
@@ -232,8 +234,8 @@ const handleIncomingTwilioMessage = async (
     return;
   }
 
-  const twilioPhoneNumber = req.body.To;
-  const userMessage = req.body.Body;
+  const twilioPhoneNumber = reqBody.To;
+  const userMessage = TwilioUtil.formatAttachments(reqBody);
```

Comment:
nit: Can we rename this (or add a comment) to make clearer that it returns a message? Something like `getMessageWithAttachments` or `formatMessageAttachments`?